### PR TITLE
Show scheduled and upcoming events, fixes #451

### LINKED ISSUE #451: WP Venues page not showing scheduled or upcoming
"Online" venues are not hyperlinked on event pages in the plugin.
image
image

This also means the 'View upcoming' / 'View scheduled' buttons on the venue list page render no results, even though there should be results.

image

​

​​

### PR DESCRIPTION
**Pull request by geeknz**

### CODE REVIEW

**Comment by ry167** [comment]
Just a note here as well, I think this will have far wider impact than the issue mentioned in #451. 
I wouldn't want to see any existing sites break. 

I could be wrong and I obviously haven't tested anything out, but the change looks scary to me.

File: includes/shortcodes/arlo-upcoming-events.php [hunk 1]
```diff
@@ -293,7 +293,7 @@ private static function generate_list_sql($atts, $import_id, $for_pagination = f
 
         $join = [];
         // We check that the event startdatetime with offset (converted to UTC) has not passed
```
Comment (phil-chan-arlo) [inline includes/shortcodes/arlo-upcoming-events.php:295]: Could we remove this comment about startdatetime if we aren't filtering for it anymore?

Comment (geeknz) [reply includes/shortcodes/arlo-upcoming-events.php]: It's been removed :)

**Review by phil-chan-arlo — APPROVED** [review]

**Review by ArloSoftwareTeam — APPROVED** [review]
Looks good.

**Pull request merged**